src/endpoints/PricingTables/types.ts

```diff
@@ -0,0 +1,63 @@
+export interface PricingTable {
+    options: Option[];
+}
+
+export enum BillingCycle {
+    MONTH = 'month',
+    YEAR = 'year',
+}
+
+export enum Currency {
+    EUR = 'eur',
+    USD = 'usd',
+}
+
+export enum OptionId {
+    STARTER = 'starter',
+    CORE = 'core',
+    PREMIUM = 'premium',
+}
+
+export enum LimitDisplayName {
+    STORIES = 'Stories',
+    USERS = 'Users',
+    SITES = 'Sites',
+    CONTACTS = 'Contacts',
+    EMAIL_SENDS = 'Email sends',
+    CUSTOM_SENDER_ADDRESSES = 'Custom sender addresses',
+}
+
+export interface Option {
+    id: `${OptionId}`;
+    display_name: string | null;
```

Comment:
It is ok to make this not null.

Reply:
https://github.com/prezly/javascript-sdk/pull/215/commits/dfd2002b381e9b832b6077669f586ea6a27ea78b